common/admin.py

```diff
@@ -74,7 +74,7 @@ class RegionAdmin(BaseRegionalAdminMixin, VersionAdmin):
     list_display = ("id", "name", "slug", "website_slogan")
 
     def has_add_permission(self, request):
-        return False
+        return request.user.is_superuser
```

Comment:
Who will have the `superuser` permissions? 🤔

Reply:
@mtrofimiuk ☝️

Reply:
Global admin.

Reply:
IMO a management command would be better. Currently, only the developer should be able to add, edit and delete regions.

Reply:
I have created story for that: https://stanwood.atlassian.net/browse/MCS-1994